src/qt/proposalwizard.cpp

```diff
@@ -0,0 +1,385 @@
+// Copyright (c) 2025 The Dash Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <qt/proposalwizard.h>
+
+#include <qt/forms/ui_proposalwizard.h>
+
+#include <qt/guiutil.h>
+#include <qt/rpcconsole.h>
+#include <interfaces/node.h>
+#include <qt/walletmodel.h>
+#include <qt/bitcoinunits.h>
+#include <qt/optionsmodel.h>
+#include <governance/object.h>
+#include <governance/validators.h>
+
+#include <QDateTime>
+#include <QDateTimeEdit>
+#include <QDoubleSpinBox>
+#include <QJsonDocument>
+#include <QJsonObject>
+#include <QLabel>
+#include <QLineEdit>
+#include <qt/qvalidatedlineedit.h>
+#include <QPlainTextEdit>
+#include <QPushButton>
+#include <QStackedWidget>
+#include <QTimer>
+
+namespace {
+// Minimal helper to encode plain ASCII JSON to hex for RPC
+static QString toHex(const QByteArray& bytes)
+{
+    return QString(bytes.toHex());
+}
+}
+
+ProposalWizard::ProposalWizard(interfaces::Node& node, WalletModel* walletModel, QWidget* parent)
+    : QDialog(parent)
+    , m_node(node)
+    , m_walletModel(walletModel)
```

Comment:
add `qt/proposalwizard.{h,cpp}` and `forms/proposalwizard.ui` to list of non-backported

Reply:
@coderabbitai, please add this type of thing to your "always check for" list of items. PRs that add files, that are not from backports / from bitcoin, must always* (there may be some limited exemptions for stuff like subtree etc) add these added files to the list of non-backported files (test/util/data/non-backported.txt)